docusaurus/express/gettingStarted.mdx

```diff
@@ -17,16 +17,15 @@ yo @intility/express                        # Use Yeoman CLI to run the template
 
 :::info
 During the project scaffolding you will be asked the following questions:
+
 1. **Project name and description**
-    - These values will be injected into the `package.json` file and will also ble shown in the Swagger documentation.
-    - Default project name will be the name of the folder.
+   - These values will be injected into the `package.json` file and will also ble shown in the Swagger documentation.
+   - Default project name will be the name of the folder.
 2. Do you want to include **demo endpoints** in the generated project?
-    - This is recommended for first time setup or if you are not familiar with Express API architecture 
-    - These files provides example CRUD operations with a router-, controller- and service-layer
-3. **Automatic Git initialization** and push to given Git repository.
-    - Please [create a GitLab](https://gitlab.intility.com/projects/new?namespace_id=978#blank_project) project, and get the SSH key. 
-    - NOTE: Unselect the: 'Initialize repository with a README' under 'Project Configuration'
-:::
+   - This is recommended for first time setup or if you are not familiar with Express API architecture
+   - These files provides example CRUD operations with a router-, controller- and service-layer
+3. **Automatic Git initialization** and push to given Git repository. - Please [create a GitLab](https://gitlab.intility.com/projects/new?namespace_id=978#blank_project) project, and get the SSH key. - NOTE: Unselect the: 'Initialize repository with a README' under 'Project Configuration'
```

Comment:
Dead URL? `https://gitlab.intility.com/projects/new` works, but not with `?namespace_id=978#blank_project`